packages/app/src/components/SearchResultsCard/style.css

```diff
@@ -13,6 +13,33 @@
   background-color: white;
 }
 
+.search-result-span {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+}
+
+.search-exit {
+  width: 300%;
+  display: flex;
+  flex-grow: 1;
+  justify-content: flex-end;
+}
+
+.thead {
+  width: 300%
+}
+
+.search-result-exit {
+  display: flex;
+
+}
+
+.green-link {
+  color:  #128B3E;
+  font-weight:bold;
+}
```

Comment:
I think we have another class in our global CSS file for other green links, would that suffice here?